playground/externals.yaml

```diff
@@ -0,0 +1,35 @@
+Components:
+  ESMA_env:
+    remote: git@github.com:GEOS-ESM/ESMA_env.git
+    tag: v1.4.0
+    local: ./@env
+  ESMA_cmake:
+    remote: git@github.com:GEOS-ESM/ESMA_cmake.git
+    tag: v1.0.10
+    local: ./@cmake
+    Components:
+      ecbuild:
+        branch: feature/FindMKL-portability-improvement
+        local: ./@cmake/@ecbuild
+        remote: git@github.com:GEOS-ESM/ecbuild.git
```

Comment:
We should articulate the advantages of the nesting here.   Thus far it merely indicates a dependency between ESMA_cmake and ecbuild.   Or is it maybe useful just to guarantee the order of commits?

Reply:
Rather than the order of commits, nesting can guarantee the order of checkout which is helpful when mounting a component inside another (in this case ecbuild is mounted inside cmake)

Reply:
One more thought.   It would be nice if we could do something akin to config interpolation so that the  "git@github.com:GEOS-ESM" part of the URL is only specified once.